### ISSUE TITLE
Feature Request (FO4VR support): improve turning

### ISSUE THREAD

**Issue by kozian** [matzman666/OpenVR-InputEmulator#41]
Right now you need to move stick to desired position and click to turn (rotate).
The idea is to autoclick each time stick gets to 1.0 or -1.0 position in horizontal direction, so you don't need to click it manually.

**Comment by Grum30** [comment]
That's right, for now it's necessary to press the stick, at the same time we turn to one of the sides to make an angle turn of X degrees (45º / 60 / º90º).

This is done by the stick in the right motion controller for Oculus Cv1 + touch controllers.

It seems to me a great idea, to work on this, to release the unnecessary and uncomfortable click in order to make the turns simply move the stick to one side.

Additionally, I would like to have the possibility of making complete smooth turns, if this were possible to complete this fantastic tool.
Probably, these advances could be at the same time of great help for other future games and controllers (Oculus Cv1 and Windows Mixed Reality headsets)
regards
